.github/workflows/sentry.yml

```
@@ -2,24 +2,66 @@ name: New Sentry Release
+      - name: Cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Yarn Install
+        run: yarn install --immutable
+
+      - name: Extract Version
```

actionlint (1.7.9)

step must run script with "run" section or run action with "uses" section

(syntax-check)